wcd2reshare.py

```diff
@@ -55,6 +60,21 @@ def lambda_handler(event: dict, context: dict) -> dict:
     Reshare / Vufind. Return an object representing an HTTP redirect response to
     ReShare / Vufind, using the query string if one was built.
     """
+    logger = logging.getLogger(__name__)
+    logging.basicConfig(level=logging.INFO)
+    env = os.environ["WORKSPACE"]
+    if sentry_dsn := os.getenv("SENTRY_DSN"):
```

Comment:
We ned to move sentry configuration to the top of the module too. When testing this locally in Docker, the exception didn't get sent to Sentry until the second time I posted to the lambda invocation URL. My guess about why is that sentry needs to be configured in the lambda runtime environment _before_ the `lambda_handler` function is called, which is why it works the second time but not the first: it gets configured during the first execution of the lambda function and then remains configured for subsequent executions. If you move this code block to the top of the module, it will get configured as soon as this module is imported to the lambda runtime, which happens in AWS behind the scenes before the function is ever executed.